.github/workflows/check.yml

```
@@ -112,6 +112,30 @@ jobs:
           python-version: "3.11"
           cache: pip
           cache-dependency-path: setup.py
-      - run: pip install -e .
+
+      # Run on current branch
+      # We CD so that "typeshed" doesn't appear in the error path
+      - run: |
+          pip install -e .
+          cd typeshed
+          flake8 typeshed --exit-zero --color never --output-file errors_new.txt
+      # Run on target branch
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ github.base_ref == 'main' }}
```

We have yet to change the default branch name at flake8-pyi (we should probably change the default branch name)

```suggestion
          ref: ${{ github.base_ref == 'master' }}
```